.travis.yml

```diff
@@ -1,49 +1,12 @@
 language: python
-sudo: false
-cache:
-  directories:
-    - $HOME/.cache/pip
-    - $HOME/virtualenv/python$TRAVIS_PYTHON_VERSION.9
-    - $HOME/moto-s3
-python:
-  - "2.7"
-env:
-  global:
-    - CFLAGS=-O0
-    - COUCHDB_USER=""
-    - COUCHDB_PW=""
-    - TRAVIS_INSTALL="y"
-  matrix:
-    - MATRIX_TYPE=javascript BOWER=yes
-    # need node installs for app_manager.XpathValidatorTest
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunnerCatchall BOWER=yes NODE=yes
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunner0
+sudo: required
```

Comment:
Is it not possible to use docker on travis with the new container-based infrastructure (`sudo: false`)?